src/Cities.jsx

```diff
@@ -0,0 +1,14 @@
+const cities = [
+  {
+    name: 'San Francisco',
+    latitude: 37.7853,
+    longitude: -122.41669,
+  },
```

Comment:
This is okay, but putting this in a JSON and then directly importing it is definitely nicer :)